pkg/network/ebpf/c/protocols/http2/decoding.h

```diff
@@ -63,28 +171,52 @@ static __always_inline bool parse_field_literal(struct __sk_buff *skb, skb_info_
         headers_to_process->type = kNewDynamicHeaderNotIndexed;
     }
     headers_to_process->original_index = index;
-    headers_to_process->new_dynamic_value_offset = skb_info->data_off;
+    headers_to_process->new_dynamic_value_offset = pktbuf_data_offset(pkt);
     headers_to_process->new_dynamic_value_size = str_len;
     headers_to_process->is_huffman_encoded = is_huffman_encoded;
     // If the string len (`str_len`) is in the range of [0, HTTP2_MAX_PATH_LEN], and we don't exceed packet boundaries
     // (skb_info->data_off + str_len <= skb_info->data_end) and the index is kIndexPath, then we have a path header,
     // and we're increasing the counter. In any other case, we're not increasing the counter.
     *interesting_headers_counter += (str_len > 0 && str_len <= HTTP2_MAX_PATH_LEN);
```

Comment:
This comment still references `skb_info`, it needs to be updated. 

Additionally, it seems to explain and list what the code until there does check, and restates what the previous comments in the function already explained. One should be able to understand all of that by just reading the code, so I am not sure what is the added value of this comment.

I suggest listing the conditions in function's comment, and maybe have the inner comments reference those conditions instead, if the code is not clear enough for it to be obvious.

Reply:
moved to the function's documentation